File: 1347-minimum-number-of-steps-to-make-two-strings-anagram/1347-minimum-number-of-steps-to-make-two-strings-anagram.js
var minSteps = function(s, t) {
    let count = new Array(26).fill(0);

    for (let i = 0; i < s.length; i++) {
        count[s.charCodeAt(i) - 97]++;
    }

    for (let i = 0; i < t.length; i++) {
        count[t.charCodeAt(i) - 97]--;
    }

    let steps = 0;

    for (let i = 0; i < 26; i++) {
        if (count[i] > 0) {
            steps += count[i];
        }
    }

    return steps;
};

File: 1347-minimum-number-of-steps-to-make-two-strings-anagram/1347-minimum-number-of-steps-to-make-two-strings-anagram.java
class Solution {
    public int minSteps(String s, String t) {
        int[] count = new int[26];

        for (char ch : s.toCharArray()) {
            count[ch - 'a']++;

        }
        for (char ch : t.toCharArray()) {
            count[ch - 'a']--;
        }
        int steps = 0;
        for (int f : count) {
            if (f > 0) {
                steps = steps + f;

            }
        }
        return steps;

    }
}